tokens-studio/figma-plugin
# 2.0: Make variable creation work across multiple files

Fixes https://github.com/tokens-studio/figma-plugin/issues/2023

This PR enables creation of variables that reference other variables part of other files. 
Also fixed an issue with variable creation from sets, where there'd be duplicate variables
Renamed the function that's responsible for creating variables based on sets to something more understandable

Had to comment out a few lines part of the `syncVariables` and syncStyles files - but didnt want to do the brunt work of removing those functions from the plugin in this PR. will do that in a followup.

## Testing this change
- Create tokens and try to create from a set - try it in multple sets as well (something that was skipped in prior PRs it seems)
- Create themes and tokens that reference other tokens in other themes, and have those create in multiple files and notice references will work

https://github.com/tokens-studio/figma-plugin/assets/4548309/17a65a99-ad37-4aad-8dca-048f2e86ad53

https://github.com/tokens-studio/figma-plugin/assets/4548309/daeedabc-f8f4-4a62-bb13-065eafdd1e87
